boards/renesas/ek_ra8m1/ek_ra8m1_defconfig

```diff
@@ -16,3 +16,5 @@ CONFIG_CONSOLE=y
 CONFIG_BUILD_OUTPUT_HEX=y
 CONFIG_BUILD_NO_GAP_FILL=y
 CONFIG_CLOCK_CONTROL=y
+
+CONFIG_FLASH=y
```

Comment:
Is it okay to just add support for EK-RA8M1?
(EK-RA8T1 and more?)

Reply:
I added support for EK-RA8D1 and MCK-RA8T1 boards. Please take a look. Thanks